server/runtime/pom.xml

```diff
@@ -467,6 +478,35 @@
                      </target>
                   </configuration>
                </execution>
+               <execution>
```

Comment:
I don't think this is required because the `server` module does the upload. I'm getting the following zips uploaded:
```
[pedro@pedro-desktop download]$ ls -1
infinispan-server-15.0.0-SNAPSHOT-src.zip
infinispan-server-15.0.0-SNAPSHOT-src.zip.sha1
infinispan-server-15.0.0-SNAPSHOT.zip
infinispan-server-15.0.0-SNAPSHOT.zip.sha1
infinispan-server-runtime-15.0.0-SNAPSHOT-src.zip
infinispan-server-runtime-15.0.0-SNAPSHOT-src.zip.sha1
```

Reply:
`infinispan-server-runtime-15.0.0-SNAPSHOT-src.zip` only contains the sources for the  `server/runtime` modules while `infinispan-server-15.0.0-SNAPSHOT-src.zip` contains all the server sources (`core`, `server/*`, etc)